WORKSPACE

```diff
@@ -66,8 +70,13 @@ raze_deps()
 # load and instantiate `rules_foreign_cc_dependencies`.
 #load("@cargo_raze//:transitive_deps.bzl", "cargo_raze_transitive_deps")
 #cargo_raze_transitive_deps()
-load("@rules_foreign_cc//foreign_cc:repositories.bzl", "rules_foreign_cc_dependencies")
-rules_foreign_cc_dependencies()
+# TODO(dmcardle) Delete comments after raze_deps above?
```

Comment:
Not sure what to do with these lines, but I think that they're unnecessary now that I added line 53-54.